<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_MidiClients</class>
 <widget class="QDialog" name="Dialog_MidiClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>427</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Midi Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30,30,30);
border-color: rgb(255, 236, 23);
color: rgb(255, 236, 170);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget_inputClients">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>280</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="treeWidget_outputClients">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>280</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QCustomPlot" name="widget_midi_connection_lines" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>95</width>
     <height>370</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30,30,30);
border-color: rgb(255, 236, 23);
color: rgb(255, 236, 170);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_midiConnectionsConnect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>395</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_midiConnectionsDisconnect">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>395</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_midiConnectionsDisconnectAll">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>395</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Disconnect All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_midiConnectionsRefresh">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>395</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_midiConnectionsExpandAll">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>395</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);
color: rgb(39, 39, 39);
disabled-background-color: rgb(40,40,40);</string>
   </property>
   <property name="text">
    <string>Expand All</string>
   </property>
  </widget>
  <zorder>widget_midi_connection_lines</zorder>
  <zorder>buttonBox</zorder>
  <zorder>treeWidget_inputClients</zorder>
  <zorder>treeWidget_outputClients</zorder>
  <zorder>pushButton_midiConnectionsConnect</zorder>
  <zorder>pushButton_midiConnectionsDisconnect</zorder>
  <zorder>pushButton_midiConnectionsDisconnectAll</zorder>
  <zorder>pushButton_midiConnectionsRefresh</zorder>
  <zorder>pushButton_midiConnectionsExpandAll</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_MidiClients</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_MidiClients</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
